Code/Tools/AssetProcessor/native/AssetManager/assetProcessorManager.cpp

```diff
@@ -194,11 +194,6 @@ namespace AssetProcessor
                  (status == AssetProcessor::AssetScanningStatus::Stopped))
         {
             AssetProcessor::StatsCapture::EndCaptureStat("AssetScanning");
-            // place a message in the queue that will cause us to transition
-            // into a "no longer scanning" state and then continue with the next phase
-            // we place this at the end of the queue rather than calling it immediately, becuase
-            // other messages may still be in the queue such as the incoming file list.
-            QMetaObject::invokeMethod(this, "FinishAssetScan", Qt::QueuedConnection);
```

Comment:
This seems like an unrelated change, or did this cause issues with the new logic?

Reply:
This is part of the optimization for the catalog setup.  Since it's using the file state cache now, I had to reorganize some of the startup stuff, which included moving this line of code.  Previously the completion of the asset scan caused AP to immediately move to the next stage, but now AP collects the scan results and waits for catalog & builder init to finish before moving on to actually process the scan results.